apps/modernization-ui/src/providers/UserContext.tsx

```diff
@@ -48,7 +48,7 @@ const UserContextProvider = ({ initial, children }: Props) => {
             // delete cookies
             document.cookie = 'nbs_user=; Max-Age=0; path=/;';
             document.cookie = 'nbs_token=; Max-Age=0; path=/;';
-            window.location.href = '/nbs/logout';
+            window.location.href = '/goodbye';
```

Comment:
No, this has to redirect to the `/nbs/logout` page in order to log out the user with Keycloak and NBS6.  The issue is that the session has timed out in NBS6 so when you go to that page you are redirected to the timeout page which is correct